src/core/lib/surface/call.cc

```diff
@@ -99,8 +99,14 @@ struct batch_control {
   } completion_data;
   grpc_closure start_batch;
   grpc_closure finish_batch;
-  gpr_refcount steps_to_complete;
+  grpc_core::Atomic<uintptr_t> steps_to_complete;
```

Comment:
Does this belong in a separate PR as well?

Reply:
Yes, in the PR that Yash alluded to above.

Reply:
Done.